### ISSUE TITLE
Looking for additional maintainer(s)!

### ISSUE THREAD

**Issue by xomiamoore** [camunda-community-hub/pyzeebe#381]
Hello all, we are searching for another maintainer or two for Pyzeebe! Please note our [maintainer and reviewer expectations.](https://github.com/camunda-community-hub/community/blob/main/maintainer-reviewer-expectations.md)

Please reply if you are interested :)

**Comment by yuval9313** [comment]
I've attempted to solve some issues but it seems no active maintainers are available and the original @JonatanMartens is revoked. May you suggest solution other then create another pyzeebe?

**Comment by xomiamoore** [comment]
Hi @yuval9313. The only maintainer is @JonatanMartens (who still has Maintainer access, where are you seeing that it is revoked?)

I agree that I'd rather not fork pyzeebe, which is why we're looking for another Maintainer. Is that something you'd be able to take on?

**Comment by dimastbk** [comment]
Hi! I would like to be maintainer of pyzeebe. Problem is that my English is not very good, so I have some problems writing docs.

**Comment by xomiamoore** [comment]
@dimastbk Hi Dmitriy! Glad to hear it, I'll give you Maintainer permissions now! 🚀 

No worries about your English, we are a global community so we have many contributors where English isn't their first language. If you ever want a proofreader, feel free to tag me and I am happy to review! :)

**Comment by dimastbk** [comment]
@xomiamoore thanks!

**Comment by dimastbk** [comment]
@xomiamoore could you remove these actions from the branch ruleset?
- unit-test (3.6)
- unit-test (3.7)

Now supported python 3.8+

**Comment by yuval9313** [comment]
Hey @xomiamoore I could certainly try

**Comment by dimastbk** [comment]
@xomiamoore and could you see MR rules? Now review from @JonatanMartens is required (see #397).

**Comment by dimastbk** [comment]
@xomiamoore no, I can't merge #398

![image](https://github.com/camunda-community-hub/pyzeebe/assets/3132181/3ebe8264-9d0c-4407-a37f-e33b1fd2bf9d)

**Comment by dimastbk** [comment]
@xomiamoore Hi! Could you see https://github.com/camunda-community-hub/pyzeebe/pull/398#issuecomment-2113964308 ?

**Comment by xomiamoore** [comment]
@dimastbk I saw you were able to merge it, did closing and reopening give you the right permissions?

**Comment by dimastbk** [comment]
@xomiamoore No, I made a hack: I added empty jobs for python3.6 and python3.7 to workflow, see https://github.com/camunda-community-hub/pyzeebe/pull/398/commits/e9bf785eeace61712b172ab78517bedcdad74a73. But we should remove this jobs from branch ruleset, because of EOL of these python versions were in 2021 and 2023.

**Comment by dimastbk** [comment]
@xomiamoore @JonatanMartens Hi! Can you add `PYPI_TOKEN` (https://github.com/camunda-community-hub/pyzeebe/blob/master/.github/workflows/publish.yml#L26) to environment variables? As I see on pypi, @JonatanMartens is owner of  https://pypi.org/project/pyzeebe/. I would like to create new prerelease https://github.com/camunda-community-hub/pyzeebe/releases

**Comment by JonatanMartens** [comment]
Hey @dimastbk @xomiamoore, I think Camunda should create a pypi organization. Let me know when it is set up and I will transfer pyzeebe to the organization.

The PYPI_TOKEN environment variable is a github secret, which I don't have access to change currently.

**Comment by JonatanMartens** [comment]
I would also like to add some maintainers to the read-the-docs. @xomiamoore let me know who I can add.

**Comment by xomiamoore** [comment]
@JonatanMartens, I got this notice when I tried to sign up for a pipy org, so not sure how long it will take to get a Camunda organization. Until then, I can add you as an admin to the repo to add the token as a secret? 

<img width="816" alt="Screen Shot 2024-06-11 at 4 40 01 PM" src="https://github.com/camunda-community-hub/pyzeebe/assets/43454823/40ee1019-1375-4996-945d-9e691371cf48">

@dimastbk would you be interested in being added to the read-the-docs as a maintainer as well?

**Comment by dimastbk** [comment]
@xomiamoore yes, [dima_st_bk](https://readthedocs.org/profiles/dima_st_bk/)

**Comment by JonatanMartens** [comment]
@xomiamoore feel free to add me as an admin, will add the token asap

**Comment by xomiamoore** [comment]
@JonatanMartens done!

**Comment by JonatanMartens** [comment]
Added the token

**Comment by xomiamoore** [comment]
@JonatanMartens Awesome, did you add @dimastbk to read-the-docs also?

**Comment by JonatanMartens** [comment]
@xomiamoore Yes

**Comment by dimastbk** [comment]
@JonatanMartens hi. Can you send invite again? I missed the link and it has expired.

**Comment by felicijus** [comment]
Hello,
I am interessed, @xomiamoore could you link [maintainer and reviewer expectations](https://github.com/camunda-community-hub/community/blob/main/maintainer-reviewer-expectations.md) again the link is broken.

**Comment by xomiamoore** [comment]
Hey @felicijus -- thanks for the heads up, it's here now: [Maintainer and reviewer expectations](https://github.com/camunda-community-hub/community/blob/main/maintainers-reviewers/maintainer-reviewer-expectations.md )

@dimastbk, @JonatanMartens, do you need another maintainer?

**Comment by dimastbk** [comment]
@xomiamoore Hi. Yes.

**Comment by dimastbk** [comment]
@JonatanMartens Hi. Can you send invite to RTD again? I missed the link and it has expired.
